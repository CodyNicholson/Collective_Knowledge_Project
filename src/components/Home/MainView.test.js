//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import { MainView } from './MainView';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

test('Tags module test', () => {
  const component = renderer.create(
    <MemoryRouter>
    	<MainView>
    	</MainView>
    </MemoryRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});