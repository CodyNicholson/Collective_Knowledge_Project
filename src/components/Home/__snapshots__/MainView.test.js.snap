// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `
<div
  className="col-md-9"
>
  <div
    className="feed-toggle"
  >
    <ul
      className="nav nav-pills outline-active"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href=""
          onClick={[Function]}
        >
          Global Feed
        </a>
      </li>
    </ul>
  </div>
  <div
    className="article-preview"
  >
    Loading...
  </div>
</div>
`;
